### ISSUE TITLE
api.py:144: UserWarning: No faces were detected. warnings.warn("No faces were detected.")

### ISSUE THREAD

**Issue by Tesrocha22** [alievk/avatarify-python#508]
The Avatarify App is functioning fine, but i get this response and display when i enter run_windows.bat command. 
I'm new to this.

C:\Users\efeos\miniconda3\envs\avatarify\lib\site-packages\face_alignment\api.py:144: UserWarning: No faces were detected.
  warnings.warn("No faces were detected.")
C:\Users\efeos\miniconda3\envs\avatarify\lib\site-packages\torch\nn\functional.py:3385: UserWarning: Default grid_sample and affine_grid behavior has changed to align_corners=False since 1.3.0. Please specify align_corners=True if the old behavior is desired. See the documentation of grid_sample for details.
  warnings.warn("Default grid_sample and affine_grid behavior has changed "
C:\Users\efeos\miniconda3\envs\avatarify\lib\site-packages\torch\nn\functional.py:1639: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")

![Screenshot (1)](https://user-images.githubusercontent.com/123034008/213412763-47b2e324-83cd-47ab-8880-82bd592a3390.png)

**Comment by JohanAR** [comment]
duplicate of #507
